OpenAPI/kiota/query-parameters-request-headers.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Query Parameters and Request Headers
+description: Learn how to use Query Parameters and Request Headers to customize API calls.
+author: baywet
+ms.author: vibiret
+ms.topic: conceptual
+date: 08/01/2023
+---
+
+# Query Parameters and Request Headers
+
+Kiota API clients provide a fluent API you can use to customize request before sending them out to the service.
+
+## Query Parameters
+
+Kiota will generate an additional type for each operation to reflect the query parameters in the API description.
+
+The application code can then use that type to customize the request and benefit from compile time validations.
+
+Consider the following sample:
+
+```csharp
+var result = await todoClient.TaskLists["taskListId"].ToDos["todoId"].AssignedTo.GetAsync(x => x.QueryParameters.Select = new string[] { "id", "title" });
+```
+
+The URL template for the requestBuilder will look something like `{+baseurl}/taskLists/{task_list_id}/toDos/{todo_id}/assignedTo{?select,expand}`.
+
+And the resulting URI will be `https://contoso.com/taskLists/taskListId/toDos/todoId/assignedTo?select=id,title`
+
+> [!NOTE]
+> [Enums in Query parameters](https://github.com/microsoft/kiota/issues/2306) are encoded with open `String`.
```

Comment:
I'm struggling with this phrase "are encoded with open `String`".

- Why the capital `String`?
- What is an "open" string?
- Who encodes it?

Do we really just mean here that if there's a property that is of an enum type, you can't pass the enum value itself in code, you have to use its string name?

Reply:
thanks for making the update, resolving